web/app.js

```diff
@@ -1939,6 +1939,22 @@ const PDFViewerApplication = {
     }
     addWindowResolutionChange();
 
+    if (typeof PDFJSDev === "undefined" || !PDFJSDev.test("MOZCENTRAL")) {
+      let mode;
+      switch (AppOptions.get("viewerCssTheme")) {
+        case 1:
+          mode = "is-dark";
+          break;
+        case 2:
+          mode = "is-light";
+          break;
+      }
+
+      if (mode) {
+        document.documentElement.classList.add(mode);
+      }
+    }
+
```

Comment:
Placing this code inside of a method called `bindWindowEvents` feels quite strange, and you probably want to run this a lot sooner anyway to reduce the risk of the light-theme "flashing" by on load.

The old code would run approximately here: https://github.com/mozilla/pdf.js/blob/ffbfd680ec418c05afdcafb6b3e6e962f978f8b0/web/app.js#L230

Reply:
Oh yes I forgot to move this code.